src/uShip.Logging.Tests/LoggerTests.cs

```diff
@@ -137,6 +137,22 @@ public void should_format_message_properly_for_graphite_counter_with_subkey()
             log.Received().Info(expectedValue);
         }
 
+        [Test]
+        public void should_format_message_properly_for_count_method_with_subkey()
+        {
+            var logFactory = Substitute.For<LogFactory>();
+            var log = Substitute.For<ILog>();
+            logFactory.Create(ConfiguredLogger.Graphite).Returns(log);
+
+            var logger = new Logger(logFactory, Substitute.For<LoggingEventDataBuilder>());
+            logger.Count(GraphiteKey.Test.ToString(), "SubKey", 1);
+
+            var hostName = Environment.MachineName;
+            var expectedValue = String.Format("graphite.test.uShip.Logging.Tests.GraphiteKey.SubKey~source={0}:1|c", hostName);
```

Comment:
Noticed that converting to string for the `Count` method produces extra information not present when the `GraphiteKey` is passed directly as it is on the other tests.  Wasn't clear why so I thought I would raise it in case this would be a concern for any reason.